seatunnel-connectors-v2/connector-pulsar/src/main/java/org/apache/seatunnel/connectors/seatunnel/pulsar/config/SourceProperties.java

```diff
@@ -167,8 +174,20 @@ public class SourceProperties {
                     .longType()
                     .noDefaultValue()
                     .withDescription("Stop from the specified epoch timestamp (in milliseconds)");
+    public static final Option<String> FORMAT =
+            Options.key("format")
+                    .stringType()
+                    .noDefaultValue()
```

Comment:
No defaule value? Or the default value is `json`? I suggest define the default value in option.  `public static final String DEFAULT_FORMAT = "json";` is not a good way.